terraform/modules/private-endpoints/bastion/main.tf

```diff
@@ -65,15 +65,15 @@ resource "azurerm_subnet_network_security_group_association" "bastion_subnet_ass
 }
 
 resource "azurerm_public_ip" "public_ip" {
-  name                = "bastion-ip-${var.resource_uid}"
+  name                = "pip-bastion-${var.resource_uid}"
   location            = var.location
   resource_group_name = var.resource_group_name
   allocation_method   = "Static"
   sku                 = "Standard"
 }
 
 resource "azurerm_bastion_host" "bastion" {
-  name                = "bastion-host-${var.resource_uid}"
```

Comment:
Let's use `bas-` for the [Bastion host resource](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations#networking).